src/bot-handlers/initialization.js

```diff
@@ -0,0 +1,143 @@
+// Helpers
+const { SUPPORTED_CONTRACT_VERSIONS, PublicNetworks } = require('@uma/common');
+
+// UMA JS libs
+const {
+  FinancialContractClient,
+  Networker,
+  Logger: logger,
+  createReferencePriceFeedForFinancialContract,
+  multicallAddressMap,
+} = require('@uma/financial-templates-lib');
+
+// Contract ABIs and network Addresses.
+const { findContractVersion } = require('@uma/core');
+const { getAbi } = require('@uma/contracts-node');
+
+const {
+  getJsonRpcUrl,
+} = require('forta-agent');
+
+const Web3 = require('web3');
+
+// initialize global constants, web3 gets populated on initialization
+const getTime = () => Math.round(new Date().getTime() / 1000);
+const web3 = new Web3(new Web3.providers.HttpProvider(getJsonRpcUrl()));
+
+// Returns whether the Financial Contract has expired yet
+async function checkIsExpiredOrShutdown(financialContractClient) {
+  const { financialContract, contractType } = financialContractClient;
+  const [expirationOrShutdownTimestamp, contractTimestamp] = await Promise.all([
+    contractType === 'ExpiringMultiParty'
+      ? financialContract.methods.expirationTimestamp().call()
+      : financialContract.methods.emergencyShutdownTimestamp().call(),
+    financialContract.methods.getCurrentTime().call(),
+  ]);
+  // Check if Financial Contract is expired.
+  if (
+    Number(contractTimestamp) >= Number(expirationOrShutdownTimestamp)
+    && Number(expirationOrShutdownTimestamp) > 0
+  ) {
+    return true;
+  }
+  return false;
+}
+
+// takes in a list of financial contract address and price feed config and returns a
+// list of { financialContractClient, priceFeed }
+async function processContractAndPriceFeed({ financialContractAddress, priceFeedConfig }) {
+  // find contract version
+  const detectedContract = await findContractVersion(financialContractAddress, web3);
+
+  // Check that the version and type is supported.
+  // Note if either is null this check will also catch it.
```

Comment:
Uber nit: Consistency with first character in comments, uppercase vs. lowercase.

Reply:
+1 I think we've got a mix since we were pulling code from outside repos, but other files in here stick to lowercase. I suggest we go lowercase